cli/server/server.go

```diff
@@ -504,11 +504,11 @@ func startServer(ctx *cli.Context) error {
 	if err != nil {
 		return cli.NewExitError(err, 1)
 	}
-	rpcServer := server.New(chain, cfg.ApplicationConfiguration.RPC, serv, oracleSrv, log)
 	errChan := make(chan error)
+	rpcServer := server.New(chain, cfg.ApplicationConfiguration.RPC, serv, oracleSrv, log, errChan)
 
 	go serv.Start(errChan)
```

Comment:
It's good that we provide `errChan` to RPC server constructor, but currently RPC server performs writes to `errChan` and doesn't perform reads. `serv` also doesn't read from `errChan`. I think, it should be fixed.

Reply:
I've just noticed that we handle errors in a separate Main loop, so it's OK. But https://github.com/nspcc-dev/neo-go/pull/2445#discussion_r857334777 is still relevant.